README.md

```diff
@@ -108,28 +208,90 @@ Please be sure to set up both parameters accordingly to your entities name.
+        empty your cache and do a hard refresh on your browser.
+
+-   To filter the conversations, use the filters on the left side of the
+    screen. Alternatively, you can filter the conversations by clicking
+    a keyphrase on a summary card.
+
+-   To search for a specific topic, type what you want to find into the
+    search bar, for example, if you want to find conversations where the
+    customer is staying at the Ritz, you can type \"ritz\" or \"ritz
+    hotel\" and the relevant conversations will surface.
+
+-   To view a full conversation, click anywhere on the summary card.
+    This will bring up the timeline of the conversation. You will also
+    be able to view the full transcription and the meta data for the
+    conversation. In this view you can search through this specific
+    conversation using the search bar and by clicking on the keyphrases.
+
+-   To view the relationships between topics, click the \'View entity
+    map\' button. Here you will choose a Facet and number of nodes and
+    be able to view the relationships between the phrases.
+
+### How to make updates to the UI
+
+Using the options on the customize page, you can upload your own logo
+and set the background and text colors for the navigation bar. For
+information on how to make other customizations, refer to the UI
+documentation.
+
+-   From the home page, click on the customize button.
+
+![image](/images/readMe/image13.png)
+
+![image](/images/readMe/image14.png)
+
+-   Select the rectangle next to background color to use the color
+    picker to select a new background color for the header.
+
+-   Select the rectangle next to text color to use the color picker to
+    select a new footer text color.
+
+![image](/images/readMe/image15.png)
+
+-   Select the choose file button to upload a new image.
+
+![image](/images/readMe/image16.png)
+
+Click save to apply the updates.
+
+**More Documentation**
+
+-   [Solution
+    Architecture](https://github.com/rturknett/Customer-Service-Conversational-Insights-with-Azure-OpenAI-Services/blob/readMe-updates/SolutionArchitecture.md)
 
-Once the deployment is completed, start processing your Bot telemetry by exporting in the "conversational-km-raw" container in the "storage account" in your "Resource Group". 
-To navigate the Web Ui, check the "App Service" resource in your "Resource Group".
+-   [Resource Group Walkthrough](https://github.com/rturknett/Customer-Service-Conversational-Insights-with-Azure-OpenAI-Services/blob/readMe-updates/SolutionArchitecture.md#resource-group-walkthrough)
 
-OPTIONAL:
-You can also download a sample dataset [here](https://csciblob.blob.core.windows.net/deploy/sample-data-unprocessed.zip)
-Unzip it and upload it in the "conversational-km-raw" container in the "storage account" in your "Resource Group".
+-   [Updating the UI](https://github.com/rturknett/Customer-Service-Conversational-Insights-with-Azure-OpenAI-Services/tree/readMe-updates/GBB.ConversationalKM.WebUI)
```

Comment:
Is UI deployment to Azure container registry documented somewhere?

Reply:
We just state in troubleshooting since in a public repo any changes you make to the repo will automatically be updated by github actions

Reply:
That's good, but if a user of the accelerator wants to make changes to the UI for a customer, how would we suggest they do it? They won't be modifying the main repo.

Reply:
If a user wants to make edits to this repo they should be using their public GitHub accounts to fork the main repo & utilize github actions. I think this makes the most sense since if you were to do this with MSFT dev account, 1 click deployment wouldn't work along with github actions. Or are you saying we need to provide instructions for people using dev accounts?

Reply:
No, I guess I'm just unclear how a user would deploy the UI from a cloned repo using GitHub actions. Our cloned repo doesn't appear to have any GitHub actions. If someone clones the repo, and then clicks the "Deploy to Azure" button, are you thinking it will automatically deploy the UI to a new container registry (not Fabrizio's)?

Reply:
So, we currently have the capability to enable GitHub actions in our current repo we just haven't enabled it. So we could write something up about enabling it, but seemed like Fabrizio thought it was straight forward.  If someone clones the repo and clicks deploy they would still point to our container, however the second they make changes to the github repo and push it to github, a new container specified by githubSecrets would update with a new UI image. (All from the web-ui.yml). I definitely see the confusion as this would only work with github actions enabled and github secrets storing your container configuration.

Reply:
That makes sense. Have we tried this? I'm still deploying the UI from my local environment. I do think some brief guidance on how to deploy changes to the UI in the Web UI README would be helpful.

Reply:
Never tried this since Fabrizio said it worked and I thought CI/CD was out of scope. What guidance do you think would help?